app/assets/javascripts/movies.js

```diff
@@ -0,0 +1,19 @@
+$(function() {
+  $('html').on('change', 'select#movie_event_ids.chosen', function(){
```

Comment:
I'm not sure I understand, I used event delegation with the [.on](https://api.jquery.com/on/) method to make sure the `change` event is only fired when the `select#movie_event_ids.chosen` element is changed. 
Can the event be fired even if I don't see requests on the network console?